dist/index.js

```diff
@@ -6191,12 +6191,12 @@ async function maybeInstallRebar3(rebar3Spec) {
 
 async function getOTPVersion(otpSpec0, osVersion) {
   const otpVersions = await getOTPVersions(osVersion)
-  const otpSpec = otpSpec0.match(/^(OTP-)?([^ ]+)/)
+  const otpSpec = otpSpec0.match(/^(OTP-|maint-)?([^ ]+)/)
   let otpVersion
   if (otpSpec[1] && !isStrictVersion()) {
     throw new Error(
       `Requested Erlang/OTP version (${otpSpec0}) ` +
-        "should not contain 'OTP-'",
+        "should not contain 'OTP-, or maint-'",
```

Comment:
To be sure I'm following along here, we're saying if the string you provided ends up matching two groups (`OTP-` or `maint-`) and version number, but you did not specify use strict versioning, we throw, is that all correct?